frontend/packages/console-shared/src/components/badges/DevPreviewBadge.tsx

```diff
@@ -1,8 +1,14 @@
 import * as React from 'react';
 import { Label } from '@patternfly/react-core';
+import './Badge.scss';
 
 const DevPreviewBadge: React.FC = () => (
-  <Label style={{ backgroundColor: '#D93F00' }}>Dev Preview</Label>
+  <Label
+    className="odc-preview-badge"
+    style={{ backgroundColor: '#D93F00', borderRadius: 'var(--pf-global--BorderRadius--sm)' }}
```

Comment:
Include these styles in the css class

Reply:
Those styles when added in the class seem to be getting overriden by patternfly.

Reply:
How about using `!important` for these styles in the scss?

Reply:
Thank! Using `!important` worked.